src/roaring.c

```diff
@@ -81,6 +82,35 @@ roaring_bitmap_t *roaring_bitmap_create_with_capacity(uint32_t cap) {
     return ans;
 }
 
+
+void roaring_bitmap_add_ordered(roaring_bitmap_t *r, size_t n_args, const uint32_t *vals) {
+
+    if (n_args == 0) {
+        return;
+    }
+    if (n_args == 1) {
+        roaring_bitmap_add(r, vals[0]);
+        return;
+    }
+
+    if (n_args < 1024 || r->high_low_container.size > 0) {
```

Comment:
I do not entirely understand this condition. Why do we need `r->high_low_container_size > 0`? And why is 1024 used as threshold and not some other value?

Reply:
Sorry about this `n_args < 1024`, it shouldn't be there.
`r->high_low_container.size > 0` is because this implementation only works when the roaring_bitmap is empty